narwhals/utils.py

```diff
@@ -1059,3 +1059,9 @@ def generate_repr(header: str, native_repr: str) -> str:
         "| Use `.to_native` to see native output |\n└"
         f"{'─' * 39}┘"
     )
+
+
+def check_column_exists(columns: list[str], subset: str | list[str] | None) -> None:
```

Comment:
doesn't subset need to be a list here?
```suggestion
def check_column_exists(columns: list[str], subset: list[str] | None) -> None:
```

Reply:
looks like this is wrong in `_spark_like` actually - could you correct that one from
```
subset: str | list[str] | None = None,
```
to
```
subset: list[str] | None = None,
```
please?